wporg-gp-translation-events.php

```diff
@@ -163,16 +163,15 @@ function submit_event_ajax() {
 		wp_send_json_error( 'Nonce verification failed', 403 );
 	}
 	// This is a list of slugs that are not allowed, as they conflict with the event URLs.
-	$invalid_slugs = array( 'new', 'edit', 'attend', 'my-events' );
-	if ( isset( $_POST['event_title'] ) && in_array( sanitize_title( wp_unslash( $_POST['event_title'] ) ), $invalid_slugs, true ) ) {
-		wp_send_json_error( 'Invalid slug', 403 );
-	}
-
+	$invalid_slugs  = array( 'new', 'edit', 'attend', 'my-events' );
 	$title          = isset( $_POST['event_title'] ) ? sanitize_text_field( wp_unslash( $_POST['event_title'] ) ) : '';
 	$description    = isset( $_POST['event_description'] ) ? sanitize_text_field( wp_unslash( $_POST['event_description'] ) ) : '';
 	$event_start    = isset( $_POST['event_start'] ) ? sanitize_text_field( wp_unslash( $_POST['event_start'] ) ) : '';
 	$event_end      = isset( $_POST['event_end'] ) ? sanitize_text_field( wp_unslash( $_POST['event_end'] ) ) : '';
 	$event_timezone = isset( $_POST['event_timezone'] ) ? sanitize_text_field( wp_unslash( $_POST['event_timezone'] ) ) : '';
+	if ( isset( $title ) && in_array( sanitize_title( $title ), $invalid_slugs, true ) ) {
```

Comment:
It has been sanitized above, there's no need to sanitize, right? Though maybe above it should be changed to `sanitize_title()` instead of `sanitize_text_field()`?

Reply:
What I want in this line is the slug, to look for it in the array, so I think is the correct function, because the title has been sanitized before.

- [sanitize_text_field](https://developer.wordpress.org/reference/functions/sanitize_text_field/). Sanitizes a string from user input or from the database.
- [sanitize_title](https://developer.wordpress.org/reference/functions/sanitize_title/). Sanitizes a string into a slug.

Reply:
Ah, I understand now, thanks 👍